examples/qclustering/minimization.py

```diff
@@ -0,0 +1,60 @@
+import math
+
+import numpy as np
+import tensorflow as tf
+from grover import grover_qc
+from oracle import create_oracle_circ
+
+from qibo import gates
+from qibo.models import Circuit
+
+
+def duerr_hoyer_algo(distances):
+    """Perform Duerr-Hoyer algorithm.
+
+    Parameters
+    ----------
+    distance : :class:`numpy.ndarray`
+        Array of distances a point to each cluster.
+        Shape=(1, k) where k is number of cluster centers.
+
+    Returns
+    -------
+    int
+        New cluster assigned for that point.
+    """
+
+    k = len(distances)
+    n = int(math.floor(math.log2(k)) + 1)
+
+    # choose random threshold
+    index_rand = np.random.choice(list(range(k)))
```

Comment:
This is not a good way if you want to generate a random integer: you can use np.random.randint or the documentation of np.random.choice suggests np.random.choice(k)
```suggestion
    index_rand = np.random.choice(k)
```

Reply:
Sure. Thanks!